app-text/dblatex/files/dblatex-0.3.10-no-inkscape-dependency.patch

```diff
@@ -0,0 +1,14 @@
+diff -Naur dblatex-0.3.10-orig/lib/dbtexmf/core/imagedata.py dblatex-0.3.10/lib/dbtexmf/core/imagedata.py
+--- dblatex-0.3.10-orig/lib/dbtexmf/core/imagedata.py	2016-10-02 14:21:16.000000000 -0700
++++ dblatex-0.3.10/lib/dbtexmf/core/imagedata.py	2017-06-25 18:26:00.302168655 -0700
+@@ -178,8 +178,8 @@
+ class SvgConverter(ImageConverter):
+     def __init__(self, imgsrc, imgdst="", docformat="", backend=""):
+         ImageConverter.__init__(self, imgsrc="svg", imgdst=imgdst)
+-        self.add_command(["inkscape", "-z", "-D", "--export-%(dst)s=%(output)s",
+-                          "%(input)s"])
++        self.add_command(["rsvg-convert", "-f", "%(dst)s", "-o","\"%(output)s\"",
```

Comment:
The point is that people still can use dblatex without installing inkscape if they want to, since inkscape is bit on the heavier side compared to librsvg.